### PR TITLE
Added new dev instance url to cli config

### PR DESCRIPTION
**Pull request by rv0lt**
## Read this before submitting the PR

1. Always create a Draft PR first
2. Go through sections 1-5 below, fill them in and check all the boxes
3. Make sure that the branch is updated; if there's an "Update branch" button at the bottom of the PR, rebase or update branch.
4. When all boxes are checked, information is filled in, and the branch is updated: mark as Ready For Review and tag reviewers (top right)
5. Once there is a submitted review, implement the suggestions (if reasonable, otherwise discuss) and request an new review.

If there is a field which you are unsure about, enter the edit mode of this description or go to the [PR template](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/.github/pull_request_template.md); There are invisible comments providing descriptions which may be of help.

## 1. Description / Summary

Added new dev instance url to cli config

## 2. Jira task / GitHub issue

[_Link to the github issue or add the Jira task ID here._](https://scilifelab.atlassian.net/jira/software/projects/DDS/boards/13?selectedIssue=DDS-2122)

## 3. Type of change

What _type of change(s)_ does the PR contain?

**Check the relevant boxes below. For an explanation of the different sections, enter edit mode of this PR description template.**

- [X] New feature
  - [ ] Breaking: _Why / How? Add info here._ 
  - [X] Non-breaking 
- [ ] Bug fix 
- [ ] Security Alert fix 
  - [ ] Package update 
    - [ ] Major version update 
- [ ] Documentation 
- [ ] Workflow 
- [ ] Tests **only** 

## 4. Additional information

- [X] [Sprintlog](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/SPRINTLOG.md) 
- [ ] Blocking PRs 
  - [ ] Merged 
- [ ] PR to `master` branch: _If checked, read [the release instructions](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/docs/procedures/new_release.md)_ 
  - [ ] I have followed steps 1-8. 

## 5. Actions / Scans

_Check the boxes when the specified checks have passed._

**For information on what the different checks do and how to fix it if they're failing, enter edit mode of this description or go to the [PR template](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/.github/pull_request_template.md).**

- [X] **Black**

- [X] **Pylint**

- [X] **Prettier**

- [X] **Yamllint**

- [X] **Tests**

- [X] **TestPyPI**

- [X] **CodeQL**

- [X] **Trivy**

- [X] **Snyk**

### CODE REVIEW

File: dds_cli/__init__.py [hunk 1]
```diff
@@ -61,12 +61,15 @@ class DDSEndpoint:
...
     BASE_ENDPOINT_REMOTE = "https://delivery.scilifelab.se/api/v1"
     BASE_ENDPOINT_REMOTE_TEST = "https://testing.delivery.scilifelab.se/api/v1"
+    BASE_ENDPOINT_REMOTE_DEV = "https://dev.delivery.scilifelab.se/api/v1"
     if os.getenv("DDS_CLI_ENV") == "development":
         BASE_ENDPOINT = BASE_ENDPOINT_LOCAL
     elif os.getenv("DDS_CLI_ENV") == "docker-dev":
         BASE_ENDPOINT = BASE_ENDPOINT_DOCKER
     elif os.getenv("DDS_CLI_ENV") == "test-instance":
         BASE_ENDPOINT = BASE_ENDPOINT_REMOTE_TEST
+    elif os.getenv("DDS_CLI_ENV") == "development-instance":
```
Comment (i-oden) [inline dds_cli/__init__.py:71]: picky but I'd prefer this: 
```suggestion
    elif os.getenv("DDS_CLI_ENV") == "dev-instance":
```
Change this in the same push as the version

**Review by i-oden — CHANGES_REQUESTED** [review]
Change the version of the CLI as well so that it's updated on testPyPI
In `version.py`: 
```
__version__ = "2.7.1-pentest"
```
or something like that.

**Comment by rv0lt** [comment]
For it to be accepted with letters is can only be "dev" or "post" keywords
https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers
> Change the version of the CLI as well so that it's updated on testPyPI In `version.py`:
> 
> ```
> __version__ = "2.7.1-pentest"
> ```
> 
> or something like that.

**Comment by i-oden** [comment]
> For it to be accepted with letters is can only be "dev" or "post" keywords https://packaging.python.org/en/latest/specifications/version-specifiers/#version-specifiers
> 
> > Change the version of the CLI as well so that it's updated on testPyPI In `version.py`:
> > ```
> > __version__ = "2.7.1-pentest"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > or something like that.

`dev` as you have now is better anyway so all good

**Review by i-oden — CHANGES_REQUESTED** [review]
We'll have to change the version again. Just undo the version change that you've done. TestPyPI doesn't work anyway and we won't make a new release today so the users will have to clone the dds-cli repo and use the dev branch anyway. 
```bash
inaod@inaod-xps-15:~/Documents/repos/DDS/dds_cli$ dds motd ls
     ︵ 
 ︵ (  )   ︵ 
(  ) ) (  (  )   SciLifeLab Data Delivery System 
 ︶  (  ) ) (    https://dev.delivery.scilifelab.se/ 
      ︶ (  )    CLI Version 2.7.1.dev0 
          ︶
INFO     No saved token found, or token has expired, proceeding with authentication                                                                                                                                            
DDS username: sdfsdf
DDS password: 
ERROR    Failed to authenticate user: Incompatible version lengths.  
```

**Comment by rv0lt** [comment]
Ok, so close the PR then?

> We'll have to change the version again. Just undo the version change that you've done. TestPyPI doesn't work anyway and we won't make a new release today so the users will have to clone the dds-cli repo and use the dev branch anyway.
> 
> ```shell
> inaod@inaod-xps-15:~/Documents/repos/DDS/dds_cli$ dds motd ls
>      ︵ 
>  ︵ (  )   ︵ 
> (  ) ) (  (  )   SciLifeLab Data Delivery System 
>  ︶  (  ) ) (    https://dev.delivery.scilifelab.se/ 
>       ︶ (  )    CLI Version 2.7.1.dev0 
>           ︶
> INFO     No saved token found, or token has expired, proceeding with authentication                                                                                                                                            
> DDS username: sdfsdf
> DDS password: 
> ERROR    Failed to authenticate user: Incompatible version lengths.  
> ```

**Review by i-oden — APPROVED** [review]

**Comment by i-oden** [comment]
@rv0lt Ok to merge

**Pull request merged**